_posts/2018-07-14-del_dotto_historic_tasting.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Del Dotto Historic Tasting and Tour"
+date:   2018-07-14 15:20:00 -0400
+categories: review tour tasting
+---
+Del Dotto ($75): http://www.deldottovineyards.com
+
+For me, Del Dotto is a solid introduction to the nicer Napa wine country experiences.  It has currently two locations (their historic site and St. Helena site), with a third coming online later this year.  My recommendation is to do the cellar barrel tasting at either of the two aforementioned sites: the historic site has a rustic feel to its chiseled-out caves, while the St. Helena site has a sophisticated, old, Italian feel.  Arrive 5-10 min before your appointment and you’ll enjoy an aperitif (likely their Sauvignon Blanc) before the tasting begins along with some nice photo opportunities.
+
+Many people haven’t experienced a barrel tasting before, which makes this a fun, unique experience.  The number of wines you get to sample and their quality/price make the $75 tasting price tag seem very reasonable, almost a steal by Napa standards.  In all, you get samples of the following wines:
+
+* Sauvignon Blanc from bottle (aperitif)
+* 2017 Pinot Noir (Cinghiale Vineyard) ($75)
+* 2016 Sangiovese ($58)
+* 2016 Cabernet Franc ($85)
+* 2016 Cabernet Sauvignon (French Oak) ($85)
+* 2016 Cabernet Sauvignon (American Oak) ($85)
+* 2016 Cabernet Sauvignon (Rutherford Estate) ($195)
+* 2016 Cabernet Sauvignon (Vineyard 887) ($195)
+* 2016 Chardonnay from bottle (Cinghiale Vineyard) ($85)
+  * Not on the tasting menu, but if you express interest, they may pour it
```

Comment:
markdown wise I *think* this should have two `**` to make it a sub-bullet. to make it clear you may want to rephrase the beginnign (`This is not on the tasting menu...`)